.github/workflows/preview.yml

```diff
@@ -1,10 +1,5 @@
 name: preview
-on:
-  pull_request:
-    branches:
-      - master
-      - develop
-      - 'release/**'
+on: [push, pull_request]
```

Comment:
This is an unrelated change to the scope of this PR please keep this separate.

Reply:
This was done temporarily to allow for other PR's to deploy preview branches while this PR got reviewed. It was done in `64c5943`, will revert prior to merge.